### PR TITLE
add op(native batch norm) | feat(atenlib)

### PR DESCRIPTION
**Pull request by xiaowuhu**

### CODE REVIEW

File: onnxscript/function_libs/torch_aten/ops/core.py [hunk 1]
```diff
@@ -4152,19 +4152,99 @@ def aten_narrow_copy(self: TensorType, dim: int, start: INT64, length: INT64) ->
...
+        running_mean = op.Squeeze(op.ReduceMean(input, axes))
+
+    if running_var is None:  # Using input var
+        mean = op.ReduceMean(input, axes)
+        input_sub_mean = op.Sub(input, mean)
+        sqr_input_sub_mean = op.Mul(input_sub_mean, input_sub_mean)
+        running_var = op.Squeeze(op.ReduceMean(sqr_input_sub_mean, axes))
+
+    # Have to split to 2 private functions, because training_function return 3 outputs
+    # While inference_function return 1 output
```
[the remark this answers is not in the document]
Comment (xiaowuhu) [reply onnxscript/function_libs/torch_aten/ops/core.py:4187]: aten function always return 3, but ONNX return 1 when inference, return 3 when training.

[the remark this answers is not in the document]
Comment (justinchuby) [reply onnxscript/function_libs/torch_aten/ops/core.py:4187]: Can we set training to the onnx op to always be true then?

[the remark this answers is not in the document]
Comment (justinchuby) [reply onnxscript/function_libs/torch_aten/ops/core.py:4187]: I assume it’s not stateful? @gramalingam

[the remark this answers is not in the document]
Comment (xiaowuhu) [reply onnxscript/function_libs/torch_aten/ops/core.py:4187]: I think we still need to identify training and inference because we will investigate the about the training on onnx-script :)
